Sources/Log.cpp

```diff
@@ -0,0 +1,38 @@
+#include "Log.h"
+
+QFile logs("log-"+ QDateTime::currentDateTime().toString() + "-" + QUuid::createUuid().toString() + ".txt");
+QTextStream logstream(&logs);
+
+void InformationToStream(const QString type, const QMessageLogContext &context, const QString &message, const QString &datetime)
+{
+    logstream << type << datetime << ": " << message << " " << context.file << " " << context.line << " " << context.function << endl;
+}
+
+void KvMessageHandler(QtMsgType type, const QMessageLogContext &context, const QString &message)
+{
+    QString datetime = QDateTime::currentDateTime().toString();
```

Comment:
So it is better instread of `InformationToStream` function just create here string for output:
```
QString message = QString("%1: %2").arg(datetime).arg(message);
```
Something like that (with more info).
Also when context has no info (when `QT_MESSAGELOGCONTEXT` is not defined) `context` variable should not be used (so log will be more readable).